django/core/checks/model_checks.py

```diff
@@ -25,6 +29,20 @@ def check_all_models(app_configs=None, **kwargs):
             )
         else:
             errors.extend(model.check(**kwargs))
+    duplicate_db_table_list = [
+        db_table_attr for db_table_attr, model_list in db_table_attr_map.items()
+        if len(model_list) > 1
+    ]
+    if duplicate_db_table_list:
+        for db_table in duplicate_db_table_list:
+            errors.append(
+                Error(
+                    "db_table '%s' was declared in multiple models: '%s'"
+                    % (db_table, ', '.join(db_table_attr_map[db_table])),
+                    obj=model,
+                    id='models.E020'
```

Comment:
This should be `models.E028` as `E020` maps to ``The `<model>.check()` class method is currently overridden.``.